### PR TITLE
Rožos aptiktuolio pakeitimai

### PR DESCRIPTION
**Pull request by AArlauskas**
Darau PR ne del to kad noriu o del to kad reikia. Atsiras laiko, gal ir noras sugris

### CODE REVIEW

**Comment by AArlauskas** [comment]
https://www.youtube.com/watch?v=Q55Ic4Mpdlk

File: app-android-xamarin/Moodis/Moodis/Feature/MainActivity.cs [hunk 1]
```diff
@@ -46,6 +57,7 @@ protected override void OnActivityResult(int requestCode, [GeneratedEnum] Result
             {
                 Finish();
             }
+
```
Comment (tmaciulis22) [inline app-android-xamarin/Moodis/Moodis/Feature/MainActivity.cs:60]: nice

**Review by tmaciulis22 — CHANGES_REQUESTED** [review]
Is login with credentials still working? Beisdes, there are some conflicts

File: app-android-xamarin/Moodis/Moodis/Feature/MainActivity.cs (hunk 1 above)
Comment (AArlauskas) [reply app-android-xamarin/Moodis/Moodis/Feature/MainActivity.cs:60]: Why shouldn't it be?

**Pull request closed without merging**